source/brailleDisplayDrivers/albatross.py

```diff
@@ -0,0 +1,822 @@
+	95: "up1",
+	96: "down1",
+	97: "left",
+	98: "up2",
+	103: "lWheelRight",
+	104: "lWheelLeft",
+	105: "lWheelUp",
+	106: "lWheelDown",
+	151: "attribute3",
+	192: "attribute4",
+	193: "f9",
+	194: "f10",
+	195: "f11",
+	196: "f12",
+	197: "f13",
+	198: "f14",
+	199: "f15",
+	200: "f16",
+	201: "home2",
+	202: "end2",
+	203: "eCursor2",
+	204: "cursor2",
+	205: "up3",
+	206: "down2",
+	207: "right",
+	208: "down3",
+	213: "rWheelRight",
+	214: "rWheelLeft",
+	215: "rWheelUp",
+	216: "rWheelDown",
+}
+# Maximum number of keys in key combination.
+MAX_COMBINATION_KEYS = 4
+# These are ctrl keys which may start key combination.
+CONTROL_KEY_CODES: List[int] = [
+	1,  # attribute1
+	42,  # attribute2
+	83,  # f1
+	84,  # f2
+	89,  # f7
+	90,  # f8
+	91,  # home1
+	92,  # end1
+	93,  # eCursor1
+	94,  # cursor1
+	151,  # attribute3
+	192,  # attribute4
+	193,  # f9
+	194,  # f10
+	199,  # f15
+	200,  # f16
+	201,  # home2
+	202,  # end2
+	203,  # eCursor2
+	204,  # cursor2
+]
+# Send this to Albatross to confirm that connection is established.
+ESTABLISHED = b"\xfe\xfd\xfe\xfd"
+# If no connection, Albatross sends continuously byte \xff followed by byte
+# containing various settings like number of cells.
+INIT_START_BYTE = b"\xff"
+# Send information to Albatross enclosed by these bytes.
+START_BYTE = b"\xfb"
+END_BYTE = b"\xfc"
+# To keep connected these both above bytes must be sent periodically.
+BOTH_BYTES = b"\xfb\xfc"
+# Display requires at least START_BYTE and END_BYTE combination within
+# approximately 2 seconds from previous appropriate data packet.
+# Otherwise it falls back to "wait for connection" state. This behavior
+# is built-in feature of the firmware of device.
+# How often BOTH_BYTES should be sent and reconnection tried
+KC_INTERVAL = 1.5
+
+
+# Controls most of read operations and tries to reconnect when needed
+# Suspended most of time.
+class ReadThread(Thread):
+	def __init__(
+			self, readFunction, disableFunction, event, dev, *args, **kwargs):
+		super().__init__(*args, **kwargs)
+		self._readFunction = readFunction
+		self._disableFunction = disableFunction
+		self._event = event
+		self._dev = dev
+
+	def run(self):
+		data = dwEvtMask = DWORD()
+		log.debug(f"{self.name} started")
+		while not self._event.isSet():
+			# Try to reconnect if port is not open
+			if not self._dev.is_open:
+				log.debug(
+					f"Calling {self._readFunction.__name__}, port {self._dev.name} not open")
+				self._readFunction()
+				if not self._dev.is_open:
+					log.debug(
+						f"Sleepin {KC_INTERVAL} seconds, port {self._dev.name} not open")
+					self._event.wait(KC_INTERVAL)
+					continue
+			try:
+				if not SetCommMask(self._dev._port_handle, EV_RXCHAR):
+					# Exiting
+					if self._event.isSet():
+						break
+					log.debug("SetCommMask failed")
+					raise(ctypes.WinError())
+				result = ctypes.windll.kernel32.WaitCommEvent(
+					self._dev._port_handle, byref(dwEvtMask),
+					byref(self._dev._overlapped_read))

```

Comment:
In general, having multiple items per line is not readable, please split these up into a separate lines

```suggestion
				result = ctypes.windll.kernel32.WaitCommEvent(
					self._dev._port_handle,
					byref(dwEvtMask),
					byref(self._dev._overlapped_read)
				)
```